### DISCUSSION TITLE
Is notifications@github.com be hacked?

### DISCUSSION THREAD

**Discussion by spartucus** [community/community#102076] [Other Feature Feedback, Questions, & Ideas]
<img width="1513" alt="image" src="https://github.com/community/community/assets/6071887/1a71ac3f-911a-4ba4-938d-4f1f38311e78">

<img width="1189" alt="image" src="https://github.com/community/community/assets/6071887/25b61c9c-651b-414d-8350-ababe03eb65d">

So many spams

**Comment by yashmalvade2002** [comment]
ask to chatgpt

**Comment by spartucus** [comment]
I've recently been notified about issues and PRs in several repos, but I haven't subscribed to these repos. From what I observed, the notifications were sent to me via the email address [notifications@github.com](mailto:notifications@github.com), where I was mentioned in the emails. This results that I was automatically subscribed to the issues and PRs.

<img width="1519" alt="image" src="https://github.com/community/community/assets/6071887/1778f22b-2a27-4b66-841c-f95a33cc2b18">

**Comment by spartucus** [comment]
<img width="1772" alt="image" src="https://github.com/community/community/assets/6071887/25d1fc77-b7b9-4c00-a661-25d7e98fe50f">
Another one, I have never subscribed to this issue, nor do I have the watch repository.
